config/database.yml

```diff
@@ -8,6 +8,7 @@ production:
   password: <%= ENV['DB_PASS'] %>
   timeout: 5000
   prepared_statements: false
+  pool: <%= ENV["DB_POOL"] || ENV['RAILS_MAX_THREADS'] || 5 %>
```

Comment:
Is canvas-lms thread safe? 
https://github.com/StrongMind/canvas-lms/blob/sm-stable/config/environments/production.rb#L58